### ISSUE TITLE
avif output ext wrong

### ISSUE THREAD

**Issue by alanwilter** [vseventer/sharp-cli#82]
If I do:

```bash
mkdir -p out_sharp_webp
time sharp --effort 4 -q 100 --format webp -i 'out_ce_png/*.png' -o out_sharp_webp/
```

It outputs:

```bash
out_sharp_webp/CP-0_0.webp
...

identify out_sharp_webp/CP-0_0.webp
out_sharp_webp/CP-0_0.webp WEBP 2576x1934 2576x1934+0+0 8-bit sRGB 579228B 0.000u 0:00.000
```
Correct.

Now if I do:

```bash
mkdir -p out_sharp_avif
time sharp --effort 4 -q 100 --format avif -i 'out_ce_png/*.png' -o out_sharp_avif/
```

I got:

```bash
out_sharp_avif/CP-0_0.png
...
```

`PNG`? But it's not PNG!

```bash
identify out_sharp_avif/CP-0_0.png
identify-im6.q16: improper image header `out_sharp_avif/CP-0_0.png' @ error/png.c/ReadPNGImage/4092.
```

I know it's `avif` but it's not getting the right extension.

**Comment by vseventer** [comment]
Thanks for reporting - turns out this is a bug specific to HEIF files.

**Comment by vseventer** [comment]
Released as v4.1.1.
